scalableminds/webknossos
# Update TimeTracking view for arbitrary date ranges

### Mailable description of changes:
- [Admin.TimeTracking] Extended the time tracking view to include more statistics and support arbitrary 31 day date ranges

### Steps to test:
- Go to `localhost:9000/timetracking`
- Select a date range and user --> See charts
- Select a date range > 31 --> error
- Try to break it

### Issues:
- fixes #2180 

------
